README.md

```diff
@@ -10,22 +10,38 @@ I wanted to make a JavaScript library named pee.js because of a typo and a pun.
 
 ## Usage
 
-### Browser (pee.js)
+This is a hybrid (commonjs and ecmascript module) package that is available on https://www.npmjs.com/package/pee.js
+```shell
+# To install the package in your project
+npm install --save pee.js
+```
+
+### Browser [ecmascript] (pee.js)
 ```js
-import { leak } from './pee.js';
+import { leak } from 'pee.js';
 leak(69); // Leaks 69 MB of memory
 ```
+See `test/pee.test.js` for a more in-depth example
 
-### Node
-https://www.npmjs.com/package/pee.js
-```console
-npm install pee.js
-```
+### Node [commonjs] (pee.cjs)
 ```js
-const pee = require('pee.js');
-pee.leak(420);
+const { leak } = require('pee.js');
+leak(420); // Leaks 420 MB of memory
+```
+See `test/pee.test.cjs` for a more in-depth example
+
+### Development
```

Comment:
some handy instructions for potential future contributors as i'd expect such an innovative project will gain traction pretty fast plus now we have exposure via the https://pee.js.org page